src/Resources/config/admin_routing.yml

```diff
@@ -59,8 +59,8 @@ sylius_refund_order_refund_payment_list:
         _sylius:
             template: "@SyliusRefundPlugin/Order/Admin/RefundPayment/list.html.twig"
             repository:
-                method: findByOrderNumber
-                arguments: $orderNumber
+                method: findByOrder
+                arguments: expr:service('sylius.repository.order').findOneByNumber($orderNumber)
```

Comment:
I'm not sure about this approach. We're calling the repository to retrieve the object, to pass it to the same repository but another function 💃 maybe we could just introduce a custom repository method that would take the order number and join the orders' table in a query builder?

Reply:
It is not the same repository, I'm calling the order repository to retrieve the order, to pass it to the refund payment repository.
However, introducing a custom repository method is ok for me, I wonder about that, but the above solution was just simpler, but I agree that it was probably the worse option due to performance (2 calls to db instead of 1 with additional join).

Reply:
Yeah, indeed, these are 2 different repositories xD Sorry, I had a hallucination 💃